internal/fields/dependency_manager.go

```diff
@@ -221,5 +229,21 @@ func transformImportedField(fd FieldDefinition) common.MapStr {
 		}
 		m.Put("fields", t)
 	}
+
+	if len(fd.MultiFields) > 0 {
+		var t []common.MapStr
+		for _, f := range fd.MultiFields {
+			i := transformImportedMultiField(f)
+			t = append(t, i)
+		}
+		m.Put("multi_fields", t)
+	}
 	return m
 }
+
+func transformImportedMultiField(fd MultiFieldDefinition) common.MapStr {
+	return common.MapStr{
```

Comment:
nit: double-checking, it isn't possible that `multi_fields` have children, right :D ?

Reply:
I don't think so :)